src/microdns.h

```diff
@@ -72,7 +72,8 @@ struct mdns_hdr {
         uint16_t num_add_rr;
 };
 
-typedef void (*mdns_callback)(void*, int, const struct rr_entry *);
```

Comment:
I think the renaming should be in a separate commit, and that probably means you need to bump some versions since this is a breaking API change

Reply:
We might worry about bumping the version out of this PR though, let's do it when this is merged

Reply:
Sounds good to me.